src/components/metadata/index.js

```diff
@@ -106,12 +107,30 @@ const MetaData = ({
             <div className="pipeline-metadata__list">
               <dl className="pipeline-metadata__properties">
                 <MetaDataRow label="Type:" value={metadata.node.type} />
-                <MetaDataRow
-                  label="Dataset Type:"
-                  visible={isDataNode}
-                  kind="type"
-                  value={metadata.datasetType}
-                />
+                {!isTranscoded && (
+                  <MetaDataRow
+                    label="Dataset Type:"
+                    visible={isDataNode}
+                    kind="type"
+                    value={metadata.datasetType}
+                  />
+                )}
+                {isTranscoded && (
+                  <MetaDataRow
+                    label="Original Type:"
+                    visible={isDataNode}
+                    kind="type"
+                    value={metadata.originalType}
+                  />
+                )}
+                {isTranscoded && (
+                  <MetaDataRow
+                    label="Transcoded Type:"
+                    visible={isDataNode}
+                    kind="type"
+                    value={metadata.transcodedType}
+                  />
+                )}
```

Comment:
This can be merged into one condition: `{isTranscoded && ... }`. Also the label should be `Transcoded Types` (note the plural).